.gitignore

```diff
@@ -0,0 +1,39 @@
+/.deps/
+/.libs/
+/INSTALL
+/Makefile
+/Makefile.in
+/aclocal.m4
+/autom4te.cache/
+/compile
+/config.guess
+/config.log
+/config.status
+/config.sub
+/configure
+/depcomp
+/doc/Makefile
+/doc/Makefile.in
+/example
+/example.o
```

Comment:
Why not ignore all *.o files? Please consider adding .gitignore in its own commit.